en_US.json

```diff
@@ -661,6 +661,7 @@
 	"edgeHostname.cancel.btn": "Cancel",
 	"edgeHostname.client.isp.cannot.contain.allandotherisp": "The selected client ISPs contain \"All\" and other ISP(s). Please select \"All\" only or chose other ISP(s).",
 	"edgeHostname.client.isp.limit.size": "Please choose up to 10 ISPs.",
+	"edgeHostname.client.isp.selected.all": "Please choose other ISP entries.",
```

Comment:
I suggest saying simply "Please choose other ISPs."

Reply:
Yes, I'll merge it after I change it like that. Thanks.